### ISSUE TITLE
CKV2_AWS_27 warning is shown even when it's fixed

### ISSUE THREAD

**Issue by fabasoad** [bridgecrewio/checkov#1651]
**Describe the bug**
Query logging should be implemented to fix `CKV2_AWS_27` warning. Based on [AWS documentation](https://aws.amazon.com/premiumsupport/knowledge-center/rds-postgresql-query-logging/) to enable it I need to define 2 parameters: `log_statement` and `log_min_duration_statement`. This is what I did in Terraform file:

```terraform
resource "aws_db_instance" "db" {
  ...
  parameter_group_name  = aws_db_parameter_group.postgres_parameters.name
}

resource "aws_db_parameter_group" "postgres_parameters" {
  name_prefix = "my_name"
  family = "postgres10"

  parameter {
    name  = "log_statement"
    value = "ddl"
  }

  parameter {
    name  = "log_min_duration_statement"
    value = 1000
  }
}
```

But anyway I have this warning:

```
Check: CKV2_AWS_27: "Postgres RDS has Query Logging enabled"
	FAILED for resource: aws_db_instance.db
	File: /my_file.tf:1-28
	Guide: https://docs.bridgecrew.io/docs/ensure-postgres-rds-has-query-logging-enabled
```

**To Reproduce**
1. Define parameters described above.
2. Run `checkov` command.

**Expected behavior**
`CKV2_AWS_27` warning shouldn't appear.

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - OS: `linux alpine` and `macOS`
 - Checkov Version: `2.0.438`

**Additional context**
N/A

**Comment by brukau** [comment]
on the policy here https://github.com/bridgecrewio/checkov/blob/master/checkov/terraform/checks/graph_checks/aws/PostgresRDSHasQueryLoggingEnabled.yaml#L22

it should connect with **aws_db_parameter_group** instead of _aws_rds_cluster_parameter_group_

**Comment by fabasoad** [comment]
@brukau yeah, it is connected with `aws_db_parameter_group` as described in description. Or you meant vice versa - should be `aws_rds_cluster_parameter_group`?

**Comment by brukau** [comment]
@fabasoad your example is good, policy should be changed to use db_parameter_group instead of cluster_parameter_group

or maybe not instead but together

basically there are two ways how Postgres can be deployed with RDS:
- `aws_db_instance` together with `aws_db_parameter_group`
- or using RDS Aurora Cluster `aws_rds_cluster` and `aws_rds_cluster_parameter_group`

**Comment by fabasoad** [comment]
So, I use `aws_db_instance` + `aws_db_parameter_group`. The problem is that `checkov` is still showing `CKV2_AWS_27` warning.
